packages/shared/dashboards/capacity-card/capacity-card.tsx

```diff
@@ -203,6 +206,21 @@ const CapacityCardRow: React.FC<CapacityCardRowProps> = ({
       </GridItem>
       <GridItem span={3} key={`${data?.name}~value`}>
         {dataUnavailable ? '-' : value}
+        {!dataUnavailable && !isPercentage && (
+          <span>
+            &nbsp; / &nbsp;
+            <Popover
+              position="right"
+              bodyContent={t(
+                'For standalone MCG clusters, there is no set limit.'
+              )}
+            >
+              <Button variant="plain" className="details-item__popover-button">
+                {t('No Limit')}
```

Comment:
why not directly this ??
```suggestion
                {t(' / No Limit')}
```

Reply:
Adding it here adds the underline/hyperlink under the "/" too, which wasn't present in the UX design :sweat_smile: .

This is why I separated the "/"